### ISSUE TITLE
Metagenotype page has no workflow progression

### ISSUE THREAD

**Issue by jseager7** [pombase/canto#1913]
Related: #1598 

The _Metagenotype Management_ page has no link or button to suggest what the user should do next, after they have created their metagenotypes and annotated them. The _Genotype Management_ page provides a 'forward' link to the _Metagenotype Management_ page, but the _Metagenotype Management_ page is a dead end: users only have the option to go back to the summary page, which could leave curators unsure of whether they've actually done everything they should've done &ndash; particularly if we're encouraging them to follow a specific workflow.

I think we should add a 'forward' navigational link that leads to another page, but there's multiple options for what this other page should be:

- **Option 1:** Link to the summary page.
  - **Advantages:** allows users to review all their annotations, and provides a more obvious link that finishes the session.
  - **Disadvantages:** does exactly the same thing as the 'back' navigational button, so it's not clear what we should label the button in this case: 'Proceed to summary' is almost the same as 'Back to summary' on the other button, which sounds odd. It's possible that the user could still miss the 'Submit to curators' button on the summary page. 

- **Option 2:** Link to the 'Curation finished' page (the same destination as the 'Submit to curators' button on the summary page).
  - **Advantages:** allows users to finish the curation session as quickly as possible.
  - **Disadvantages:** doesn't give the user any way to review all of their existing annotations, since they don't return to the summary page. This might not be a problem if the user has been checking their annotations throughout, but this might not be likely, since the other workflow links don't send the user back to the summary page either.

- **Option 3:** Link to a new 'review annotations' page, which lists all annotations in the session, and provides a final 'forward' link to submit to curators.
  - **Advantages:** allows users to review all their annotations without having any unnecessary information to distract them. Should make the workflow particularly explicit if we provide another 'forward' link on this page.
  - **Disadvantages:** requires more work to implement a new template. If the 'forward' navigational link is placed below the annotations, then it might be easy to miss the link when there are lots of annotations to scroll through (however, this also forces the user to at least skim-read their annotations).

- - -

Another related problem is that the 'Make metagenotype' button (see below) looks like a 'forward' navigational link, but doesn't behave as such: it creates a metagenotype and keeps the user on the same page. This button should be presented differently, so it's not confused with the new navigational button we're planning to add.

![metagenotype_links](https://user-images.githubusercontent.com/37659591/58871901-c80f5d80-86ba-11e9-83b4-bc9449a58ab3.PNG)

**Comment by ValWood** [comment]
Alayne and I were going to look at this when we chatted, but we spent the session on the ontology and didn't do this. We will do it next time.
Val

**Comment by kimrutherford** [comment]
> The Genotype Management page provides a 'forward' link to the Metagenotype Management page, 

Only in PHI mode.  :-)

Both management pages are confusing in their own way but I agree that the metagenotype management page is of a dead end.

I think we need to make it easy for new users.  In particular it would help if it was obvious that making an annotation was a good next step after making a (meta)genotype.  I'm not worried about users finding the button for submitting their session because they will have seen the session summary page and are likely to be able to find their way back to it.

Maybe after the user makes their first (meta)genotype we should display a pop-up or a message pointing them at the "Start a ... annotation .." link in case they are stuck?

@ValWood and @CuzickA do you have an opinion?

**Comment by ValWood** [comment]
Yes I think it needs to be really clear. I haven't done a paper from Scratch. I have been working through the ones Alyne is doing. At some point I will do the genotype entry part and see if I have any other suggestions, but I think your current idea can't hurt...

**Comment by jseager7** [comment]
> Maybe after the user makes their first (meta)genotype we should display a pop-up or a message pointing them at the "Start a ... annotation .." link in case they are stuck?

I hadn't considered this, but I agree that this is probably more important than the forward link. I think a message would be less intrusive than a popup, particularly when the user has done a curation session before (in which case they already know what to do).

If this idea works well, maybe we could extend it to other parts of the application where the next step might not be obvious.

**Comment by kimrutherford** [comment]
> maybe we could extend it to other parts of the application where the next step might not be obvious.

Yep!  It would probably help a lot to have a short message after each new action.  It would have been good to do that years ago but better late than never.

> We could disable it in advanced mode if it annoys experienced curators.

Yep, we'd need to do that.

**Comment by ValWood** [comment]
Needs user testing

**Comment by CuzickA** [comment]
> @CuzickA have any of the PHI-base curators ever complained about the fact that the Metagenotype page doesn't link forward to any other page? Or has it been obvious enough to them that after creating and annotating their metagenotypes they should go back to the Summary page and then submit their session?

@jseager7, I don't recall this being an issue with any of the PHI-base curators.
